### ISSUE TITLE
Please update uriparser to 0.9.0, includes security fixes

### ISSUE THREAD

**Issue by hartwork** [Linuxbrew/homebrew-core#10033]
Hi!

Just a quick heads up that [uriparser 0.9.0](https://github.com/uriparser/uriparser/releases/tag/uriparser-0.9.0) with security fixes has been released.

Thanks for updating!

**Comment by iMichka** [comment]
Yes, there is some workflow: in 99% of the cases, formulae get updated in the mac repository first. Once this is done, I (or somebody else, but mainly me at the moment) will merge the changes to the linux repo and build linux binaries. This can sometimes take 1 day or 2, depending on my workload.

Some formulae are linux only. For these we update them directly on this repo.

There is no concept of "maintainer" for a formula: everybody can open a pull request and bump the version in the case there is a new one. We highly rely on our community to open these requests because it is sometimes hard to keep track of the 4000 and more formulae we ship.

**Comment by hartwork** [comment]
Thanks for your words on the process.
FYI homebrew has an update now, please see https://github.com/Homebrew/homebrew-core/pull/33501.
